code/modules/multiz/fall_hit.dm

```diff
@@ -36,7 +36,10 @@
 /mob/living/fall_impact(var/turf/landing)
 	var/gravity = get_gravity()
 	var/total_brute_loss = 0
-	if(gravity > 0.5)
+	var/obj/item/airbag/airbag = locate() in contents
+	if(mind && mind.suiciding)
+		airbag = null
```

Comment:
Not liking this. How does the airbag (automatic) know this jump is suicidal in nature? Make it so it opens anyway, but you die of a heart attack during the fall.

Reply:
instead of doing a costly locate then nulling it, just don't locate it if you don't need it

Reply:
fixed it